### PR TITLE
[GIBS] Remove intro page and update root URL

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
We have a React app currently at `/education/gi-bill/post-9-11/ch-33-benefit/`. That intro page is being moved into Drupal as a typical tool intro page with a CTA widget.  This means we no longer need the content of the intro page stored in React. The Drupal page and these changes will be launched together.

However, we also need to do some reconfiguring of the React app's rootUrl and routing. So now,

`/education/gi-bill/post-9-11/ch-33-benefit/status` -> `rootUrl` of app
`/education/gi-bill/post-9-11/ch-33-benefit/print` -> moved to `/education/gi-bill/post-9-11/ch-33-benefit/status/print` so that that client-side routing can function

Must launch with https://github.com/department-of-veterans-affairs/devops/pull/6560
Issue - https://github.com/department-of-veterans-affairs/va.gov-team/issues/6756

## Testing done
1. Created a test page in local vagov-content at `/education/gi-bill/post-9-11/ch-33-benefit/index.md` to imitate a Drupal page
1. Ran the app and API locally
1. Navigated to the app and stepped through its page to confirm I didn't break anything.

## Screenshots

<details>

<summary>Status page</summary>

image

</details>

<details>

<summary>Print page (notice updated URL)</summary>

image

</details>

## Acceptance criteria
- [x] Intro page lives in Drupal
- [x] Client-side routing isn't broken

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by ncksllvn** [comment]
Note - CI is expected to fail until `/education/gi-bill/post-9-11/ch-33-benefit/` is published, but we can't publish that change until this PR is otherwise ready to go.

File: src/applications/post-911-gib-status/manifest.json [hunk 1]
```diff
@@ -30,6 +30,10 @@
       {
         "path": "education/gi-bill/post-9-11/ch-33-benefit/",
         "name": "Check Post-9/11 GI Bill benefits"
+      },
+      {
+        "path": "education/gi-bill/post-9-11/ch-33-benefit/",
```
Comment (rianfowler) [inline src/applications/post-911-gib-status/manifest.json]: this is a duplicate path.

**Review by rianfowler — COMMENTED** [review]
couple of questions 

- What is a "typical tool intro page?" Most of the introduction pages for apps are in react. 
- Is this something intended to be used in the forms system?

**Comment by rianfowler** [comment]
couple of other questions?
- should we document this as a pattern that people should use on the platform? 
- what widget is this using?

**Comment by ncksllvn** [comment]
> What is a "typical tool intro page?" Most of the introduction pages for apps are in react.

It's a page that introduces the tool and generally contains a CTA widget leading into it, independent of how the app is actually built. It's necessitated here because other tool intro pages have side bars, and this needs one too. Examples -

https://staging.va.gov/health-care/secure-messaging/
https://staging.va.gov/health-care/refill-track-prescriptions/
https://staging.va.gov/records/get-veteran-id-cards/vic/
https://staging.va.gov/change-direct-deposit/

**Comment by ncksllvn** [comment]
> should we document this as a pattern that people should use on the platform?

That shouldn't really be necessary, because by default tool intro pages will be written in Drupal with a CTA widget attached. This app was a special case because as an older app it predates the Drupal work. But here's an example of a recently-deployed app that has a tool intro page leading into it, https://www.va.gov/disability/view-disability-rating/.

> what widget is this using?

This is a follow-up to https://github.com/department-of-veterans-affairs/vets-website/pull/11888. The widget defined there is `POST_911_GI_BILL_STATUS_WIDGET`.

**Review by rianfowler — APPROVED** [review]
thanks for the info

**Pull request merged**